package/doc/sphinx/source/documentation_pages/selections.rst

```diff
@@ -54,6 +54,7 @@ packages with the help of :ref:`Selection exporters`.
         True
 
 
+    For further details on ordered selections, see :ref:`ordered-selections-label`.
```

Comment:
[This](https://github.com/MDAnalysis/mdanalysis/runs/5807270775?check_suite_focus=true#step:5:1434) message may suggest a blank line after.

Reply:
codecov/project failed .